Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM golang:alpine as builder
+FROM golang:1.15-alpine AS builder
 
-WORKDIR /go/src/github.com/blendle/pg2kafka
-ADD . ./
+# needed for gcc
+RUN apk add --no-cache build-base && \
+    apk add --no-cache git && \
+    apk add --no-cache --upgrade bash
 
-RUN apk --update --no-cache add git alpine-sdk bash
-RUN wget -qO- https://github.com/edenhill/librdkafka/archive/v0.11.4-RC1.tar.gz | tar xz
-RUN cd librdkafka-* && ./configure && make && make install
-RUN go get github.com/golang/dep/cmd/dep && dep ensure -vendor-only
-RUN go build -ldflags "-X main.version=$(git rev-parse --short @) -s -extldflags -static" -a -installsuffix cgo .
+WORKDIR /build
+COPY . .
+
+RUN go build -tags musl -ldflags "-X main.version=$(git rev-parse --short @) -s -extldflags -static" -a -installsuffix cgo .
 
 FROM scratch
-LABEL maintainer="Jurre Stender <jurre@blendle.com>"
-COPY sql ./sql
-COPY --from=builder /go/src/github.com/blendle/pg2kafka/pg2kafka /
+
+LABEL maintainer="Ioannis Sermetziadis<ioannis@codebgp.com>"
```

Comment:
Let's add a generic codebgp email like `oss@codebgp.com`, if you agree. I am not a fun of having dev specific emails here and there.

Reply:
I agree. I will create a `dev@codebgp.com` as an alias which points to your and my email (and to other devs as we grow our team). You can use that one for now.

Reply:
Created, dev@codebgp.com is active

Reply:
However, please use dev[at]codebgp[dot]com for bot-robustness :)